packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -711,7 +712,16 @@ type SnapControllerArgs = {
    * @returns The feature flags.
    */
   getFeatureFlags: () => DynamicFeatureFlags;
+
+  /**
+   * Get the cryptographic functions to use for the client. This may return an
+   * empty object to fall back to the default cryptographic functions.
+   *
+   * @returns The cryptographic functions to use for the client.
+   */
+  getClientCryptography: () => CryptographicFunctions;
```

Comment:
Does it need to be a hook here? Couldn't we just take the `CryptographicFunctions` directly?